### DISCUSSION TITLE
Cant open sqlite db in mounted discs

### DISCUSSION THREAD

**Discussion by DantesRevengeReloaded** [dbeaver/dbeaver#23378] [General]
I have a db stored in a mounted disc, my os is ubuntu 23.10 and version it says cant open db, i changed permssions but still the problem persists, if i put the copy of db in my main disc it opens without problems, also the same db opens from my windows device, the dbeaver version is Version 24.0.1.202403241413. Also when i try to create a new db in other mounted discs again it has access denied message. the partition of disc is ext4. Ty

**Comment by mkgrgis** [comment]
@DantesRevengeReloaded , please carefully step by step check all read rights for your user by path to SQLite file on your mounted disk via `ls -l $adr` from `/` to your file. Also, if you need any data modification, please check write rights on the file and  directory of the file.
